src/libraries/sendtohelixhelp.proj

```diff
@@ -187,6 +187,20 @@
         <TestTimeout>$(_workItemTimeout)</TestTimeout>
         <LaunchTimeout>$(_workItemTimeout)</LaunchTimeout>
       </XHarnessAppBundleToTest>
+
+      <!-- Create work items for run-only apps -->
+      <XHarnessAppBundleToTest Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveRoot)runonly', '*.app', System.IO.SearchOption.AllDirectories))" >
+        <Targets Condition="'$(TargetArchitecture)' == 'arm'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'arm64'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x64'">ios-simulator-64</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x86'">ios-simulator-32</Targets>
+
+        <LaunchTimeout>$(_workItemTimeout)</LaunchTimeout>
+        <!-- The sample app doesn't need test runner -->
+        <IncludesTestRunner>false</IncludesTestRunner>
+        <!-- The sample's C# Main method returns 42 so it should be considered by xharness as a success -->
+        <ExpectedExitCode>42</ExpectedExitCode>
```

Comment:
it would be nice if we could avoid hardcoding the exit code value here e.g. by writing the value into a some file as part of the sample build and then reading it here.

that can be done in a separate PR though.

Reply:
I'll consider it in the follow-up PR, thanks!